employees/templates/employees/common/filter_option.html

```diff
@@ -0,0 +1,13 @@
+<div class="btn-group">
+  <button type="button" class="btn btn-primary button-with-white-border">{{ filter_name.value }}</button>
+  <button type="button" class="btn btn-primary button-with-white-border dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+    <span class="glyphicon glyphicon-menu-down"></span>
+    <span class="sr-only"></span>
+  </button>
+  <div class="dropdown-menu">
+      <a class="dropdown-item" href="#" style="color:rgb(44, 119, 188);font-weight: bold;">{{ filter_name.value }} number 1</a><br/>
```

Comment:
All styles should be defined in `.css` file.

Reply:
Done